File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printHex(int row);

int main(void)
{
    int getHeight = get_int("Height: ");
    // For row by row
    int row = 0;
    for (row = 1; row <= getHeight; row++)
    {
        // for each whitespace in row on left hand side
        int numOfWhitespace = getHeight - row;
        for (int i = numOfWhitespace; i > 0; i--)
        {
            printf(" ");
        }
        printHex(row);
        printf(" ");
        printf(" ");
        printHex(row);
        printf("\n");
    }
}

void printHex(int row)
{
    for (int k = row; k > 0; k--)
    {
        printf("#");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while (True):
    getHeight = get_int("Height: ")
    if not (getHeight >= 1 and getHeight <= 8):
        continue
    else:
        break

whitespace = 0
for i in range(1, getHeight+1):
    whitespace = getHeight - i
    # left side of pyramid
    print((" " * whitespace)+("#" * i), end='')
    # middle 2 whitespaces
    print("  ", end='')
    # right side of pyramid
    print("#" * i)